### PR TITLE
94 Icons Place Page Hordynskyi

### PR DESCRIPTION
**Pull request by Viktor-Hordynskyi**
image
You need to open some place and check to the following buttons:
- comment button: show you the number of reviews, and if you press it - redirect to the review section.
- average rating of the current place
- wish button, when you press it this place add/delete from your "plans list", if you are not logged in, this button will be inactive.

### CODE REVIEW

**Review by Dmitry528 — APPROVED** [review]
1) I signIn, add new place to wishList, then delete it, and between this check field under image on place page, and it was okay,
2) SignOut and this field was disabled
Good work

**Review by Dmitry528 — COMMENTED** [review]
Good work! really

File: src/components/app/app.js
Comment (marrybbb) [inline src/components/app/app.js:87]: - Button for adding places doest work. places are not added to wish list.
- I think this size of buttons are too big for single page.
- Why did you decide to change view of buttons: in the very beginning there were 5 stars and now only 1, why did you write"With to visit", we used a looltip for this.
- Blue button wont fit for new design.
- I wasnt registered but button wasnt disabed(only Dubno Castle)
image

**Review by andrii-kondratiuk — APPROVED** [review]
Tested. Everything works. Checkbox correctly disabled for guest users.

File: src/components/app/app.js
Comment (Viktor-Hordynskyi) [reply src/components/app/app.js:87]: - Three people checked - everything works well, I think the problem in your side, check your branches version and use git pull, also you need the last version of the server and log in to the application.
- In the dev branch, I don't see any changes in design, so I created these buttons as I like.
- because our average rating has floating-point value, and my decision has a sense
- it's not my work, I created a button for the previous version of App.

File: src/components/single-place/main-section/add-to-wish-list-bttn/add-to-wish-list-bttn.js
Comment (marrybbb) [inline src/components/single-place/main-section/add-to-wish-list-bttn/add-to-wish-list-bttn.js:17]: bad naming of variables. It is unclear what "find" means without context.
findPlace or smth like this better

File: src/components/app/app.js
Comment (marrybbb) [reply src/components/app/app.js:87]: - you saw changed on Andriy`s branch so you have to create smth that will fit for it.Because in any case it shoul be fixed after merging .
- What is not your work?That the  but button wasnt disabed

Comment (Viktor-Hordynskyi) [reply src/components/app/app.js:87]: I don't care what I saw. New design in dev - no, so stop spam here.
Your question about the blue color of the button, I answered that "it's not my work, I created a button for the previous version of App".
All your questions/comment have no sense just spam on GIT.

Comment (marrybbb) [reply src/components/app/app.js:87]: Ok.

Comment (marrybbb) [reply src/components/app/app.js:87]: It is not a SPAM, I ask questions according your work because it is my work. According rules of scrum and our process we should approve requests in the queue, Andriy request was opened earlier and it will be closed earlier so new design will be in dev before you.

Comment (Viktor-Hordynskyi) [reply src/components/app/app.js:87]: Ok ^_^

**Review by marrybbb — APPROVED** [review]

**Pull request merged**
